### PR TITLE
[ADF-3681] Fixing skipped tests

### PR DESCRIPTION
**Pull request by marouanbentaleb**
**Please check if the PR fulfills these requirements**

> - [ ] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [x] Other... Please describe:
Fixing the skipped automated tests

**What is the current behaviour?** (You can also link to an open issue here)

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:
https://issues.alfresco.com/jira/browse/ADF-3681

### CODE REVIEW

File: e2e/pages/adf/contentServicesPage.ts [hunk 1]
```diff
@@ -283,6 +323,13 @@ export class ContentServicesPage {
         return this;
     }
 
+    startUploadFile(fileLocation) {
```
Comment (cristinaj) [inline e2e/pages/adf/contentServicesPage.ts]: Isn't this method the same with uploadFile() method?

Comment (marouanbentaleb) [reply e2e/pages/adf/contentServicesPage.ts]: Oh yes, I was just trying another approach which needed "checkUploadButton()" to be removed from the upload method. I forgot and kept it.

File: e2e/content-services/upload/uploader_component.e2e.ts [hunk 2]
```diff
@@ -202,9 +206,8 @@ describe('Upload component', () => {
         contentServicesPage.checkContentIsNotDisplayed(pdfFileModel.name);
     });
 
-    xit('[C272792] Should be possible to cancel upload of a big file using dialog icon', () => {
-        browser.executeScript(' setTimeout(() => {document.querySelector("#adf-upload-dialog-cancel-all").click();' +
-            'document.querySelector("#adf-upload-dialog-cancel").click();  }, 3000)');
+    fit('[C272792] Should be possible to cancel upload of a big file using row cancel icon', () => {
```
Comment (cristinaj) [inline e2e/content-services/upload/uploader_component.e2e.ts]: please change 'fit' with 'it'.

File: e2e/content-services/upload/uploader_component.e2e.ts [hunk 3]
```diff
@@ -213,24 +216,23 @@ describe('Upload component', () => {
         contentServicesPage.checkContentIsNotDisplayed(largeFile.name);
     });
 
-    xit('[C260169] Should be possible to cancel upload of a big file through the cancel uploads button', () => {
+    fit('[C287790] Should be possible to cancel upload of a big file through the cancel uploads button', () => {
```
Comment (cristinaj) [inline e2e/content-services/upload/uploader_component.e2e.ts]: please change 'fit' with 'it'.

**Pull request merged**